plugins/joe_sandbox/komand_joe_sandbox/actions/submit_sample/action.py

```diff
@@ -35,6 +39,11 @@ def run(self, params={}):
             raise Exception(
                 'Unable to decode base64 input for "cookbook". ' "Contents of the file must be encoded with base64!"
             )
+        webids = None
+        if filename:
+            sample_tuple = (filename, sample_bytes)
```

Comment:
No need to change it now, but for future, there's no need to include the `tuple` part of the variable name - we know it's a tuple by looking at it :) Just `sample` would suffice.